lib/elixir/lib/access.ex

```diff
@@ -181,19 +181,20 @@ defmodule Access do
   @callback get(term :: t, key, default :: value) :: value
 
   @doc """
-  Invoked in order to access the value under `key` and updated it at the same time.
+  Invoked in order to access the value under `key` and update it at the same time.
 
   The implementation of this callback should invoke the passed function on the
```

Comment:
Instead of "invoke the passed function on the function", it should be "the passed function with the value"?